package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@microsoft/applicationinsights-core-js",
     "author": "Microsoft Corporation",
-    "version": "1.0.0-beta.5",
+    "version": "1.0.0-beta.6",
```

Comment:
This will break most packages using core, it would be better to increase major version of beta, so we have time to fix them

Reply:
Yes, this will go to 2.0.0-beta.1